bots/twitterbot.go

```diff
@@ -197,10 +202,54 @@ func (bot *TwitterBot) HandleRegistration(username string, msg []byte) string {
 	return string(res)
 }
 
-// SendDM sends a Twitter direct message msg to the given Twitter screenname.
+// sendDM sends a Twitter direct message msg to the given Twitter screenname.
 // The sender screenname should be set to the bot's reserved Twitter handle.
-func (bot *TwitterBot) SendDM(screenname, msg string) error {
+func (bot *TwitterBot) sendDM(screenname, msg string) (*twitter.DirectMessage, error) {
 	params := &twitter.DirectMessageNewParams{ScreenName: screenname, Text: msg}
-	_, _, err := bot.client.DirectMessages.New(params)
-	return err
+	dm, _, err := bot.client.DirectMessages.New(params)
+	return dm, err
+}
+
+// deleteOldDMs deletes all prior DMs before the bot runs.
+func (bot *TwitterBot) deleteOldDMs() {
+	log.Println("[registration bot] Deleting old DMs ...")
+	params := &twitter.DirectMessageGetParams{Count: 200} // up to a maximum of 200.
```

Comment:
Why delete the last 200 messages (and not all for instance)? As you discussed with @masomel above it seems possible that `deleteRequestDMs` isn't able to delete all messages. But then, is 200 a strict upper bound (if so why)? Also, I'm not sure we would want to delete those missed messages on startup.